builder.config.json

```diff
@@ -85,105 +85,122 @@
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 1
       },
       "Core.RunFlow": {
         "name": "Core.RunFlow",
         "type": "Core.RunFlow",
         "is_interactive": false,
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 2
       },
       "Core.SetContactProperty": {
         "name": "Core.SetContactProperty",
         "type": "Core.SetContactProperty",
         "is_interactive": false,
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 2
       },
       "Core.SetGroupMembership": {
         "name": "Core.SetGroupMembership",
         "type": "Core.SetGroupMembership",
         "is_interactive": false,
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 2
       },
       "MobilePrimitives.Message": {
         "name": "MobilePrimitives.Message",
         "type": "MobilePrimitives.Message",
         "is_interactive": true,
         "is_branching": false,
-        "category":  1
+        "category":  1,
+        "menu_category": 1
       },
       "MobilePrimitives.NumericResponse": {
         "name": "MobilePrimitives.NumericResponse",
         "type": "MobilePrimitives.NumericResponse",
         "is_interactive": true,
         "is_branching": false,
-        "category":  1
+        "category":  1,
+        "menu_category": 1
       },
       "MobilePrimitives.OpenResponse": {
         "name": "MobilePrimitives.OpenResponse",
         "type": "MobilePrimitives.OpenResponse",
         "is_interactive": true,
         "is_branching": false,
-        "category":  1
+        "category":  1,
+        "menu_category": 1,
+        "divider_before": true
       },
       "MobilePrimitives.SelectOneResponse": {
         "name": "MobilePrimitives.SelectOneResponse",
         "type": "MobilePrimitives.SelectOneResponse",
         "is_interactive": true,
         "is_branching": true,
-        "category":  2
+        "category":  2,
+        "menu_category": 1
       },
       "MobilePrimitives.SelectManyResponse": {
         "name": "MobilePrimitives.SelectManyResponse",
         "type": "MobilePrimitives.SelectManyResponse",
         "is_interactive": true,
         "is_branching": true,
-        "category":  2
+        "category":  2,
+        "menu_category": 1
       },
       "SmartDevices.LocationResponse": {
         "name": "SmartDevices.LocationResponse",
         "type": "SmartDevices.LocationResponse",
         "is_interactive": true,
         "is_branching": false,
-        "category":  1
+        "category":  1,
+        "menu_category": 4
       },
       "SmartDevices.PhotoResponse": {
         "name": "SmartDevices.PhotoResponse",
         "type": "SmartDevices.PhotoResponse",
         "is_interactive": true,
         "is_branching": false,
-        "category":  1
+        "category":  1,
```

Comment:
@safydy-r Not important but can we have an enum to recognize which category number means what?  Something like 
`enum MenuCategories {
    content = 1,
    weather = 2,
    ...
}`